java/lang/missing_database_authentication.yml

```
@@ -0,0 +1,30 @@
+patterns:
+  - pattern: |
+      $<SQL_DRIVER_MANAGER>.getConnection($<_>, $<_>, $<EMPTY_STRING>)
+    filters:
+      - variable: SQL_DRIVER_MANAGER
+        detection: java_lang_missing_database_authentication_sql_driver_manager
+      - variable: EMPTY_STRING
+        regex: ('|")('|")
```

You might be able to do this instead:
```suggestion
        string_regex: \A\z
```